draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -355,104 +377,111 @@ Inter-domain physical links exist only:
 -  All the physical interfaces at inter-domain links are Ethernet
 physical interfaces.
 
-Although the new optical technologies (e.g., QSFP-DD ZR 400G) allow
-the operators to provide DWDM pluggable interfaces on the IP routers,
-the deployment of those pluggable optics is not yet widely adopted.
-The reason is that most operators are not yet ready to manage packet
-and optical networks in a single unified domain. Therefore, a unified
-use case analysis is outside the scope of this document.
+Although emerging optical technologies (e.g., QSFP-DD ZR 400G) enable
+operators to deploy DWDM pluggable interfaces on IP routers, the adoption
+of these technologies remains limited. This is primarily because most
+operators are not yet prepared to manage packet and optical networks
+within a single unified domain. Consequently, a unified use case analysis
+falls outside the scope of this document.
 
-This document analyses scenarios where all the multi-technology IP links,
-supported by the optical network, are intra-domain (intra-AS/intra-area), such as PE-BR, PE-P, BR-P, P-P IP links. Therefore the inter-domain IP links are always single-technology links supported by single-technology Ethernet
-links between physically adjacent IP routers.
+This document analyzes scenarios in which all multi-technology IP links
+supported by the optical network are intra-domain (intra-AS/intra-area),
+such as PE-BR, PE-P, BR-P, and P-P IP links. Consequently, inter-domain
+IP links are always single-technology connections, supported by
+single-technology Ethernet links between physically adjacent IP routers.
 
-Therefore, if inter-domain links between the optical domains exist,
-they would be used to support multi-domain optical services, which
-are outside the scope of this document.
+Therefore, if inter-domain links between optical domains exist, they
+would be utilized to support multi-domain optical services, which fall
+outside the scope of this document.
 
 The optical nodes within the optical domains can be either:
 
-- WDM nodes, as defined in {{?I-D.ietf-ccamp-optical-impairment-topology-yang}}, with an integrated ROADM functions and with or without integrated optical transponders;
+- DWDM nodes, as defined in
```

Comment:
I-D.ietf-ccamp-optical-impairment-topology-yang defines the 'WDM node' (not the 'DWDM node')

```suggestion
- WDM nodes, as defined in
```